apps/asap-server/src/entities/user.ts

```diff
@@ -18,7 +18,6 @@ export const userUpdateSchema = Joi.object({
   biography: Joi.string().allow(''),
   country: Joi.string().allow(''),
   city: Joi.string().allow(''),
-  orcid: Joi.string().allow(''),
```

Comment:
Have deleted these because orcid can't be updated in FE

Reply:
that's fine but how do we know orcid is no longer being passed down to the controller? Like.. I know we could test any other arbitrary property but i wonder if it's actually ignored? I guess you could extend/create a test in `user.route.test.ts` to see if it's really removed from the payload when passed further down to the controller